lib/cli.js

```diff
@@ -161,13 +161,14 @@
 CLI.prototype.GracefulShutdown = GracefulShutdown;
 
 /**
- * Adds an analytics event that gets sent right before the process exits.
+ * This feature has been deprecated. It used to add an analytics event that
+ * gets sent right before the process exits.
  * @param {String} name - The analytics event name
  * @param {*} data - The data payload
  * @param {String} type - The event type
  */
 CLI.prototype.addAnalyticsEvent = function addAnalyticsEvent(name, data, type) {
```

Comment:
<!--
  0 failure: 
  3 warning:  lib/cli.js line 1..., lib/cli.js line 1..., lib/cli.js line 1...
  
  
  DangerID: danger-id-Danger;
  File: lib/cli.js;
  Line: 170;
-->

- :warning: lib/cli.js line 170 – 'data' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: lib/cli.js line 170 – 'name' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: lib/cli.js line 170 – 'type' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)